samples/poc-subapp/README.md

```diff
@@ -1,5 +1,210 @@
 # subapp poc sample
 
+## The Concept
+
+- What is a subapp?
+
+  At its core, a subapp is just a component, and if React is used, a React component.
+
+  The goal is to not limit subapps to a framework, but at the moment React is the primary focus.
+
+- What makes a subapp special?
+
+  Electrode provide enhancements to make subapps special:
+
+  - `Code splitting` - Automatically detect subapps and configure webpack to split your JS by subapps.
+  - `Composable` - Create routes/pages that are composed of multiple subapps.
+  - `Lazy loading` - Dynamically lazy load and create multiple instances of subapps on a page.
+  - `Initial Props` - Automatically retrieve initial props before rendering subapps.
+  - `Async data fetch` - Use React suspense to enable single pass async data fetch within.
+  - `Server Side Rendering` - Independently enable server side render for each subapp.
+  - `Redux` - Automatically facilitate, initialize, and hydrate SSR data using Redux.
+  - `React Router` - Automatically setup component routing using react-router.
+  - `Hot module Reload` - Automatically support Hot Module Reload during development.
+
+## Introduction
+
+**How do I create a subapp?**
+
+- Electrode automatically look under the directory `src` for subapps.
+- To create a subapp, create a directory for it and an entry file with `subapp-` prefix, or it just `subapp.js`.
```

Comment:
Have this list be a numbered list instead of a bulleted list, because these are sequential steps.